Example/Emission/AppSetup.m

```diff
@@ -30,19 +30,17 @@ - (instancetype)initWithAmbientContext
     }
 
     BOOL useMaster = ![[KSCrash sharedInstance] crashedLastLaunch];
-    BOOL usePRBuild = [defaults boolForKey:ARUsePREmissionDefault];
+    BOOL usePRBuild = NO;
     BOOL useRNP = NO;
     BOOL isSimulator = NO;
 
-// Comment both of these to set yourself up as though you were running the beta
-
 #if TARGET_IPHONE_SIMULATOR
     isSimulator = YES;
 #endif
 
-
+    // Comment these out to set yourself up as though you were running the beta
+    usePRBuild = [defaults boolForKey:ARUsePREmissionDefault];
     useMaster = useMaster || isSimulator;
-
     useRNP = isSimulator || [defaults boolForKey:ARForceUseRNPDefault];
 
     if (usePRBuild) {
```

Comment:
^

Reply:
?

Reply:
oh yeah, it just moves the logic down - cooooool, ignore me

Reply:
Would have been so epic if we could’ve finished this discussion using only single characters!

Reply:
Maybe a headshake? Or a nod? Not quite sure how I could have expressed I was wrong, I agree in a single char

Reply:
!